### PR TITLE
Adding midterm

### PR DESCRIPTION
**Pull request by Rohansid**
Would love to get your comments on how this could have been improved.

How I want to take this assignment further:
- Take it online (local host) with erm
- Make it so that the program does not quit after an incorrect zip code, but instead gives the user unlimited attempts at entering a zip code correctly, with the same error message each time.

### CODE REVIEW

File: rohan_siddhanti/congress_person.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
+require 'pry'
+require 'rest-client'
+require 'json'
+require_relative 'lib/congressppl'
+
+puts
+puts "This program will give you all the Congress people that are associated with a certain zipcode. Please go ahead and type in a zipcode:"
+puts
+
+user_zip = gets.strip
```
Comment (sgriffis89) [inline rohan_siddhanti/congress_person.rb]: You could do an 'until' loop.  something like this:
user_input = gets.strip

until user_input == 'exit' do


  if /^\d{5}$/.match(user_input.to_s).nil?
      puts "Not a 5 digit number. Try again."
      user_input = gets.strip
  else

...bunch of stu...
end

File: rohan_siddhanti/congress_person.rb [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  exit
+end
+
+
+#if /^\d{5}$/ != user_zip
+#puts "Please enter a five digit zip code"
+#exit
+#end 
+
+url = "congress.api.sunlightfoundation.com/legislators/locate?zip=#{user_zip}&apikey=a88acef08fc24a5092ac6e71e5e7ccca"
```
Comment (sgriffis89) [inline rohan_siddhanti/congress_person.rb]: This is neat!  Good info to know.

File: rohan_siddhanti/congress_person.rb [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+puts
+puts "Here are the Congressmen and Women from the zipcode #{user_zip.to_i}"
+puts
+
+#posts.each do |post|
+ # puts post
+#end
+#Will put the congresspll with the ID, that's it
+
+times_to_loop = posts.size
```
Comment (sgriffis89) [inline rohan_siddhanti/congress_person.rb]: I'd definitely do a posts.each loop here!

**Comment by Rohansid** [comment]
Awesome, thanks for the feedback



- Rohan

**Pull request merged**